src/js/store/middleware/feature/form.js

```diff
@@ -218,6 +256,7 @@ const handleSubmit = (store, next, action) => {
 }
 
 const handleClear = (store, next) => {
+  store.dispatch(clearLocalStorage())
```

Comment:
We dispatch a `CLEAR_LOCAL_STORAGE` action in response to a `CLEAR` action, which clears the form after a successful submission.